bin/cli.js

```diff
@@ -12,39 +12,38 @@
 // License for the specific language governing permissions and limitations
 // under the License.
 
-(function() {
-    var utils          = require('../lib/utils');
-    var Async          = require('../lib/async');
```

Comment:
These whitespace changes look good to me, should we add `.editorconfig`, `.prettierrc`, and `.prettierignore` to make sure these are consistent when others are developing in this repo as a followup task?